terracumber_config/tf_files/SUSEManager-4.2-refenv-PRV.tf

```diff
@@ -0,0 +1,197 @@
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "Manager-4.2"
+}
+
+// Not really used in this pipeline, as we do not run cucumber
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+// Not really used in this pipeline, as we do not send emails on success (no cucumber results)
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results REF4.2-PRV $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results REF4.2-PRV: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-refenv-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://metropolis.mgr.prv.suse.net/system"
+}
+
+module "base" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  name_prefix       = "suma-ref42-"
+  use_avahi         = false
+  domain            = "mgr.prv.suse.net"
+  images            = ["centos7o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu2004o"]
+  mirror            = "minima-mirror.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  provider_settings = {
+    pool         = "ssd"
+    network_name = null
+    bridge       = "br1"
+  }
+}
+
+module "server" {
+  source                  = "./modules/server"
+  base_configuration      = module.base.configuration
+  product_version         = "4.2-nightly"
+  name                    = "srv"
+  monitored               = true
+  use_os_released_updates = true
+  disable_download_tokens = false
+  from_email              = "root@suse.de"
+  channels                = ["sle-product-sles15-sp1-pool-x86_64", "sle-product-sles15-sp1-updates-x86_64", "sle-module-basesystem15-sp1-pool-x86_64", "sle-module-basesystem15-sp1-updates-x86_64", "sle-module-containers15-sp1-pool-x86_64", "sle-module-containers15-sp1-updates-x86_64", "sle-manager-tools15-pool-x86_64-sp1", "sle-manager-tools15-updates-x86_64-sp1"]
+
+  provider_settings = {
+    mac = "aa:b2:92:03:00:d1"
+    memory = 8192
+  }
+}
+
+module "suse-client" {
+  source             = "./modules/client"
+  base_configuration = module.base.configuration
+  product_version    = "4.2-nightly"
+  name               = "cli-sles15"
+  image              = "sles15sp3o"
+
+  server_configuration    = module.server.configuration
+  use_os_released_updates = true
+
+  provider_settings = {
+    mac = "aa:b2:92:03:00:d2"
```

Comment:
Those are also looking bad, like shifted one, review it please.
```
host suma-ref42-pxy            { hardware ethernet aa:b2:92:03:00:d2; fixed-address 10.84.220.210; option host-name "suma-ref42-pxy";            }
```

Reply:
Yes, in this case you are right.

Reply:
Done. Problem is that I used the one for the proxy, but here we don't have a proxy. All other look OK.